### PR TITLE
Objects: typo in subtyping example (**) instead of (**.)

### PR DESCRIPTION
**Pull request by arnau**
The subtyping example uses `**` instead of `**.`. This PR fixes it.

### CODE REVIEW

**Comment by yminsky** [comment]
The change looks good, but the markdown wasn't updated.  You need to run "dune runtest; dune promote" to fix that.

**Comment by arnau** [comment]
@yminsky I'm afraid I don't understand why CI is failing. I ran `make test` locally and went well. Status 0, no observable issues. 🤔

File: book/files-modules-and-programs/README.md [hunk 1]
```diff
@@ -153,33 +153,19 @@ itself. [OCaml toolchain/dune]{.idx}
 
 ```sh dir=examples/freq-dune
 $ grep -o '[[:alpha:]]*' freq.ml | ./_build/default/freq.bc
```
Comment (yminsky) [inline book/files-modules-and-programs/README.md]: This looks wrong, and is also what the test is complaining about.  I'm not sure how this change got in here, but it looks like it should be reverted.

Comment (arnau) [reply book/files-modules-and-programs/README.md]: ugh, I have no idea. Let me fix this

**Comment by arnau** [comment]
@yminsky I removed the odd change. This piece of code depends on `grep` and mine, under macos outputs something different to what the CI and I assume an up to date Linux does.

If it helps in any way:

* macos Mojave 10.14.6
* grep (BSD grep) 2.5.1-FreeBSD
* GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)

**Comment by arnau** [comment]
Ok, I installed grep (GNU grep) 3.3 and now the output is as expected by the tests.

**Comment by yminsky** [comment]
Looking at this more, it looks like there's a bigger problem, which is that somewhere along the way as we redid the tooling, those examples got set up in a way that they don't compile.  I went and did a larger cleanup of the chapter, which includes your change, and a bunch of other small tweaks, here:

https://github.com/realworldocaml/book/pull/3201

Since that PR subsumes this one, I'm going to close this.  Thanks for pointing out the issue!

**Pull request closed without merging**